games/warzone2100/Portfile

```diff
@@ -217,34 +214,53 @@ if {$subport eq ${name}} {
 }
 
 if {$subport eq "${name}-music"} {
-    github.setup        Warzone2100 data-music \
-                        68e157c79f6db989120aad9e0b7ad6251c0bf721
-    version             0.0.0
-    supported_archs     noarch
+    set subport_version 0.0.0
     platforms           any
+    supported_archs     noarch
 
     long_description-append \
                         \n\nThis package ($subport) contains the music \
                         soundtrack files.
 
-    checksums           rmd160  1c76033dc721c8e049d6effe02d95d823d9ab212 \
-                        sha256  40cb829187464f361bdb2c0a9e36207b2e7bd19507454be60c89054f3e3b47b9 \
-                        size    141931427
+    variant standard conflicts high \
+        description {Install standard-quality music soundtrack} \
+    {
+        github.setup    Warzone2100 data-music-opus \
+                        81ae966cc68c242a1452ad1f168fb02bab82acea
+        version         $subport_version
+
+        checksums       rmd160  95f18b1b31f640842a1d085d22195f3354c6873e \
+                        sha256  1a86e9b08999b3a13c484a81a3c2dee7922a71bd243ef1e2607a66859c27189d \
+                        size    87292224
+    }
+    variant high conflicts standard \
+        description {Install high-quality music soundtrack} \
+    {
+        github.setup    Warzone2100 data-music-opus-hq \
+                        41dc5aac2afe5de04b2de2dc8cf699e8cbbae131
+        version         $subport_version
+
+        checksums       rmd160  7601e86b019e0895fd3a07cf5475b63ea239d0e8 \
+                        sha256  2afd7c59fa02749ecad718da26cb0f99c5d9103eb6c73bb0f37917d985c5ab46 \
+                        size    184129091
+    }
+
+    default_variants    +high
```

Comment:
you're making changes in this subport without changing the version, so you'll likely need to increase the "revision". 

Also changing the github hashes/checksums without increasing the versions should probably be treated as a "stealth update".

Reply:
I was actually hoping to do a stealth update, because the upstream developers added a new track to the music soundtrack. These tracks are completely optional, and if any track doesn't exist, it simply doesn't show up in the game's menu.

However, considering that I have added new variants to the subport, I suppose you're right, I need to bump the revision.

I've made the change.

Reply:
It still needs the stealth-update recipe as well for the changed checksums without updating the version; it’s described somewhere in the Wiki or you can search in other Portfiles.

Reply:
If you're talking about https://trac.macports.org/wiki/PortfileRecipes#stealth-updates, I don't think that it's applicable in this case. Since the commit hash gets appended to the distfile's name that gets distributed by GitHub, it's not really a stealth update as it is described in the wiki. I guess maybe I should have referred to what I was wanting to do as a "silent update".

Reply:
it's about the filename of the distfile that hets mirrored on the  MacPorts servers - is that changes as well (I didn't check).

Reply:
I just checked distfiles.macports.org, and yes, the distfile that gets mirrored on our servers does change as well. So that's one more reason why I think what I wanted to do would not be considered a stealth update, but rather simply a silent update. In any case, since the distfile name is different, a revbump should be sufficient.